google/resource_compute_instance_group_manager.go

```diff
@@ -472,9 +473,10 @@ func resourceComputeInstanceGroupManagerRead(d *schema.ResourceData, meta interf
 		update_strategy = "REPLACE"
 	}
 	d.Set("update_strategy", update_strategy.(string))
-	if err = d.Set("auto_healing_policies", flattenAutoHealingPolicies(manager.AutoHealingPolicies)); err != nil {
-		return fmt.Errorf("Error setting auto_healing_policies in state: %s", err.Error())
-	}
+
+	// When we make a list Removed, we see a permadiff from `field_name.#: "" => "<computed>"`. Set to nil in Read so we see no diff.
+	d.Set("version", nil)
+	d.Set("rolling_update_policy", nil)
```

Comment:
Do we need to set auto healing policies, too? @rileykarson

Reply:
Super weird. As long as it's not diffing, I'm happy. :) I don't know if investigating it right this second is the best use of your time, since I think 0.12 is changing all that code _anyways_. It may be worth digging into if 0.12 beta1 has the same problem?